src/components/grid/grid.stories.js

```diff
@@ -0,0 +1,191 @@
+    `${viewportSettings} (ii)`,
+    {
+      alignSelf: 'stretch',
+      colStart: 1,
+      colEnd: 13,
+      justifySelf: 'stretch',
+      maxWidth: '1300px',
+      rowStart: 1,
+      rowEnd: 1
+    },
+    groupID1
+  );
+
+  const item1900 = object(
+    `${viewportSettings} (iii)`,
+    {
+      alignSelf: 'stretch',
+      colStart: 1,
+      colEnd: 9,
+      justifySelf: 'stretch',
+      maxWidth: '900px',
+      rowStart: 2,
+      rowEnd: 2
+    },
+    groupID1
+  );
+
+  const item21500 = object(
+    `${viewportSettings} (i)`,
+    {
+      alignSelf: 'stretch',
+      colStart: 7,
+      colEnd: 13,
+      justifySelf: 'stretch',
+      maxWidth: '1500px',
+      rowStart: 1,
+      rowEnd: 1
+    },
+    groupID2
+  );
+
+  const item21300 = object(
+    `${viewportSettings} (ii)`,
+    {
+      alignSelf: 'stretch',
+      colStart: 1,
+      colEnd: 13,
+      justifySelf: 'stretch',
+      maxWidth: '1300px',
+      rowStart: 2,
+      rowEnd: 2
+    },
+    groupID2
+  );
+
+  const item2900 = object(
+    `${viewportSettings} (iii)`,
+    {
+      alignSelf: 'stretch',
+      colStart: 1,
+      colEnd: 9,
+      justifySelf: 'stretch',
+      maxWidth: '900px',
+      rowStart: 3,
+      rowEnd: 3
+    },
+    groupID2
+  );
+
+  const item31500 = object(
+    `${viewportSettings} (i)`,
+    {
+      alignSelf: 'stretch',
+      colStart: 1,
+      colEnd: 13,
+      justifySelf: 'stretch',
+      maxWidth: '1500px',
+      rowStart: 2,
+      rowEnd: 2
+    },
+    groupID3
+  );
+
+  const item31300 = object(
+    `${viewportSettings} (ii)`,
+    {
+      alignSelf: 'stretch',
+      colStart: 1,
+      colEnd: 13,
+      justifySelf: 'stretch',
+      maxWidth: '1300px',
+      rowStart: 3,
+      rowEnd: 3
+    },
+    groupID3
+  );
+
+  const item3900 = object(
+    `${viewportSettings} (iii)`,
+    {
+      alignSelf: 'stretch',
+      colStart: 1,
+      colEnd: 9,
+      justifySelf: 'stretch',
+      maxWidth: '900px',
+      rowStart: 1,
+      rowEnd: 1
+    },
+    groupID3
+  );
+
+  const item1Child = (
+    <Pod
+      alignTitle='left' as='primary'
+      border padding='medium'
+      podType='primary'
+    >
+        GridItem 1.
+    </Pod>
+  );
+
+  const item2Child = (
+    <Pod
+      alignTitle='left' as='primary'
+      border padding='medium'
+      podType='primary'
+    >
+        GridItem 2.
+    </Pod>
+  );
+
+  const item3Child = (
+    <Pod
+      alignTitle='left' as='primary'
+      border padding='medium'
+      podType='primary'
+    >
+        GridItem 3.
+    </Pod>
+  );
+
+  const gridItems = [
+    <GridItem responsiveSettings={ [item11500, item11300, item1900] }>{item1Child}</GridItem>,
+    <GridItem responsiveSettings={ [item21500, item21300, item2900] }>{item2Child}</GridItem>,
+    <GridItem responsiveSettings={ [item31500, item31300, item3900] }>{item3Child}</GridItem>
+  ];
+
+  return <GridContainer>{gridItems}</GridContainer>;
+};
+
+full.story = {
```

Comment:
could we use Basic story here? to make possible use already defined steps for Cypress? Thanks